index.js

```diff
@@ -64,25 +47,7 @@ module.exports = function ({ content, filename, sourceMap }) {
       + transpile(`${exportsTarget}.staticRenderFns = [${staticRenderFns}];`)
       + `\n${exportsTarget}.render._withStripped = true;`;
   }
-  const result = `${scriptPart}${compiledTemplate}`;
```

Comment:
Why are source maps removed?

PS: Sry. My computer is broken, which costs me 2 days to recover data.